rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize
```

Comment:
- [ ] **A rental belongs to a book and also belongs to a person**. To break the many-to-many relationship, this constructor should be modified so that Book and Person are set in it. Something of this sort: `def initialize(date, book, person)`

Reply:
Thank you! I will implement that.